### DISCUSSION TITLE
A Call for new Python Notebooks

### DISCUSSION THREAD

**Discussion by Tanu-N-Prabhu** [Tanu-N-Prabhu/Python#13] [Q&A]
Hey Guys. I am calling for python developers or geeks to share new python ideas, code snippets to this repository. This will help others understand more about Python Programming. Together let's make this a better community.

> Only selected notebooks will be accepted and published into the repository.

* Selected notebooks or codes will be directly committed to the repository, also will be added as a contributor.
* A contributor badge will be provided, and the names will be displayed in the Readme.md file.

> Feel free to contribute, Stay safe, and Happy Coding 2021.

**Comment by TatyOko28** [comment]
Hey, that sounds like a great initiative to foster collaboration in the Python community! Sharing new Python ideas and code snippets will help developers learn and improve their skills. I’d love to contribute and explore innovative Python projects, whether in automation, data science, web development, or AI. Could you share the repository link and guidelines for submission? Looking forward to being part of this!

**Reply by Tanu-N-Prabhu** [reply]
Hello, Thanks for replying. Sure, here is the repository link [Python Repository](https://github.com/Tanu-N-Prabhu/Python)

[LINK](https://github.com/Tanu-N-Prabhu/Python)

All you need to do is - if you have a Python code/document that you feel would help the community. Then you need to submit a Pull request, so I will then merge it to the main repository. But please make sure the code is well documented and explained. You can refer to my samples in the main repository for further details. Let me know if you need any help with anything. I am excited to colab with you.

Tanu
